sequencing/vcf_to_plink/dxapp.json

```diff
@@ -20,7 +20,7 @@
       ]
     },
     {
-      "name": "vcfidx_fn",
+      "name": "variants_vcfgztbi",
```

Comment:
Should this name end with an s for "array" of files?
"variants_vcfgztbi" -> "variants_vcfgztbis"
I'll leave this for you to make a final decision here.